apps/rush-lib/src/schemas/version-policies.schema.json

```diff
@@ -83,10 +83,14 @@
         "mainProject": {
           "description": "The main project for this version policy",
           "type": "string"
+        },
+        "shouldSkipChangelogVerification": {
+          "description": "If true, the version policy will not require changelog files.",
+          "type": "boolean"
         }
       },
       "required": ["policyName", "definitionName", "version", "nextBump"],
-      "additionalProperties": false
+      "additionalProperties": true
```

Comment:
You still want this to be `false`. From the [spec](https://json-schema.org/understanding-json-schema/reference/object.html):

>The `additionalProperties` keyword is used to control the handling of extra stuff, that is, properties whose names are not listed in the properties keyword. By default any additional `properties` are allowed.